UPGRADE.md

```diff
@@ -37,6 +37,158 @@ are terminated. In this way, Kong will serve new requests via the new
 configuration, without dropping existing in-flight connections.
 
 
+## Upgrade to `2.1.0`
+
+Kong adheres to [semantic versioning](https://semver.org/), which makes a
+distinction between "major", "minor" and "patch" versions. The upgrade path
```

Comment:
```suggestion
distinction between "major", "minor", and "patch" versions. The upgrade path
```

Reply:
This was also copied from the previous versions. Are Kong docs standardizing on the Oxford comma? :)

Reply:
At least on the docs website I think we are :)

Reply:
You're right! Fixing it!

https://github.com/Kong/docs.konghq.com/blob/master/STYLEGUIDE.md#serial-comma

Reply:
done!